src/api/app/views/webui2/webui/staging_workflows/_infos.html.haml

```diff
@@ -0,0 +1,18 @@
+.card.mb-3
+  %h5.card-header Infos
+  .card-body
+    %dl
+      %dt Empty projects:
+      %dd= projects_without_requests
+
+      -# TODO: Backlog was a link to group_show_path(title: @distribution.staging_manager), how does it work now?
```

Comment:
fair question - which brings us back to the 'big picture' discussion you guys like to avoid